packages/server/src/api/controllers/record.js

```diff
@@ -231,3 +235,55 @@ async function validate({ instanceId, modelId, record, model }) {
   }
   return { valid: Object.keys(errors).length === 0, errors }
 }
+
+function coerceRecordValues(record, model) {
+  for (let [key, value] of Object.entries(record)) {
+    const field = model.schema[key]
+    if (!field) continue
+    const mapping = Object.prototype.hasOwnProperty.call(
```

Comment:
Is `Object.prototype.hasOwnProperty.call` not only relevant when an objects prototype is null or it's been created with `Object.create()`? Is that something that you came across here?

If not it shouldn't be necessary. I'm also not sure the `parse` identity functions etc are necessary either. (See below comment)

Reply:
`Object.prototype.hasOwnProperty` - Lint made me do it!

I added a linter ignore line comment